content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -408,3 +408,82 @@ second ensure that [keys and certificates are being delivered](#keys-and-certifi
 
 If everything appears to be working so far, the next step is to verify that the right [authentication policy](/docs/tasks/security/authentication/authn-policy/)
 is applied and the right destination rules are in place.
+
+### Auto mTLS
+
+If you suspect [auto mTLS](/docs/reference/glossary#auto-mtls) is not working
+as expected, please first read the
+[documentation](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls).
+
+Auto mTLS has a known limitation with [workload level](/docs/tasks/security/authentication/authn-policy/#enable-mutual-tls-per-workload) peer authentication.
+When workload level peer authentication is used, you need to configure a corresponding
+destination rule configuration to let client sidecar send mTLS or plaintext
+traffic correctly.
+
+If you still think there's an issue, follow the instructions below for
+investigation.
+
+1. Get the client envoy cluster configuration to ensure auto mTLS is configured.
+You should see `transportSocketMatches` configured for the given Envoy cluster.
+
+    {{< text bash >}}
+    $ POD=$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})
+    $ istioctl proxy-config clusters ${POD}.foo -ojson | grep 'name.*outbound.*httpbin' -A130
+    "name": "outbound|8000||httpbin.default.svc.cluster.local",
+    ...
+    "transportSocketMatches": [
+            {
+                "name": "tlsMode-istio",
+                "match": {
+                    "tlsMode": "istio"
+                },
+                "transportSocket": {
+                    "name": "envoy.transport_sockets.tls",
+                    "typedConfig": {
+                        "@type": "type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.UpstreamTlsContext",
+                        "commonTlsContext": ...
+            }
+            {
+                "name": "tlsMode-disabled",
+                "match": {},
+                "transportSocket": {
+                    "name": "envoy.transport_sockets.raw_buffer"
+                }
+            }
+    {{< /text >}}
+
+1. Check the destination pod to see if the destination endpoint has the label
+`tlsMode` equals to `istio`.
+
+    {{< text bash >}}
+    $ POD=$(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name})
```

Comment:
nit: the title says to check the destination pod but the example command is using sleep not httpbin?

Reply:
yes that's intended since we are checking the source side EDS response. I added a sentence for clarification.